apps/shared/ResourceController.qml

```diff
@@ -107,17 +107,19 @@ Item {
 
     ConnectionHelper {
         id: connectionHelper
+        readonly property bool connected: status === ConnectionHelper.Online
+                                          || status === ConnectionHelper.Connected
```

Comment:
Needless to say but "status >= ConnectionHelper.Connected" would do the same. I think this is better still.

Reply:
Thanks; indeed. Updated.